### PR TITLE
Document spark dependencies in operator

### LINKED ISSUE #278: Document spark dependencies options in operator page
## Requirement - what kind of business use case are you trying to solve?
Provide documentation to explain how the Spark dependencies job is configured in the jaeger-operator CR.

## Problem - what in Jaeger blocks you from solving the requirement?
No documentation currently exists.

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #278

### CODE REVIEW

File: content/docs/next-release/operator.md [hunk 1]
```diff
@@ -391,6 +391,22 @@ storage:
     image: jaegertracing/jaeger-es-index-cleaner  // image of the job
 ```
 
+## Spark dependencies
```
Comment (objectiser) [inline content/docs/next-release/operator.md]: Should we change this to `Deriving dependencies`?

File: content/docs/next-release/operator.md [hunk 2]
```diff
@@ -391,6 +391,22 @@ storage:
...
+
+Spark dependencies collects spans from storage, analyzes links between services and stores them for later presentation in the UI.
```
Comment (objectiser) [inline content/docs/next-release/operator.md]: "Spark dependencies collects" -> "The processing to derive dependencies will collect ..."
"and stores" -> "and store"

File: content/docs/next-release/operator.md [hunk 3]
```diff
@@ -391,6 +391,22 @@ storage:
...
+This job is needed only with production strategy and it supports Cassandra and Elasticsearch backends.
```
Comment (objectiser) [inline content/docs/next-release/operator.md]: "This job can only be used with the `production` strategy and storage type `cassandra` or `elasticsearch`."?

File: content/docs/next-release/operator.md (hunk 1 above)
Comment (pavolloffay) [reply content/docs/next-release/operator.md]: Based on the other title names it's better to use your proposal.

**Pull request merged**